koku/api/query_handler.py

```diff
@@ -48,24 +50,21 @@ def convert_value(self, value, expression, connection):
         return value.strftime('%Y-%m-%d')
 
 
-class QueryHandler(object):
+class QueryHandler:
     """Handles report queries and responses."""
 
-    def __init__(self, query_parameters, url_data,
+    def __init__(self, query_parameters,
                  tenant, default_ordering, **kwargs):
         """Establish query handler.
 
         Args:
             query_parameters    (Dict): parameters for query
-            url_data        (String): URL string to provide order information
             tenant    (String): the tenant to use to access CUR data
             default_ordering (String) default ordering of response items
             kwargs    (Dict): A dictionary for internal query alteration based on path
         """
         LOG.debug(f'Query Params: {query_parameters}')
 
-        self.query_parameters = query_parameters
```

Comment:
Assigning the `query_parameters` instance attribute got moved to provider-specific child classes due to initialization ordering issues.